docs/pages/reference/backends.mdx

```diff
@@ -80,9 +80,13 @@ If your load balancer supports HTTP health checks, configure it to hit the
 `/readyz` [diagnostics endpoint](../management/diagnostics/monitoring.mdx) on
 machines running Teleport. This endpoint must be enabled by using the
 `--diag-addr` flag to teleport start: `teleport start
```

Comment:
I would move the `teleport start` command to a code fence, e.g.,

````
```code
$ teleport start --diag-addr=0.0.0.0:3000
```
````
Otherwise, this paragraph becomes a little jumbled.